lib/kwtool/accounts/user.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Kwtool.Accounts.User do
```

Comment:
According to [our convention](https://nimblehq.co/compass/development/code-conventions/elixir-phoenix/#business-domain), let's put this module inside the `Schema` scope.
```elixir
defmodule Kwtool.Accounts.Schema.UserSchema do
```